Languages/en.ini

```diff
@@ -10,7 +10,7 @@ PopupDismissText=Press &START; to dismiss.
 [ScreenInit]
 # UnsupportedGame is a SystemMessage displayed at startup if SM5 is in kickbox, beat, lights, etc. game
 UnsupportedGame=%s is not a playable game in Simply Love.\nPlease choose a different game.
-UnsupportedSMVersion=%s %s is not supported by Simply Love.\nPlease choose a different theme or use SM5.1-beta or SM5.0.12.
+UnsupportedSMVersion=%s %s is not supported by Simply Love.\nPlease choose a different theme or use ITGmania 0.8.0.

```

Comment:
Hmm I think it would be annoying to remember to update all the strings whenever the ITGmania version requirement bumps. I think either call a function that can determine the latest version (via SL-SupportHelpers) or rephrasing to a version agnostic messaging would be nicer.

Reply:
ended up adding an extra variable to the format string with the version